utils/swift_build_support/swift_build_support/products/benchmarks.py

```diff
@@ -51,19 +51,31 @@ def build(self, host_target):
     def should_test(self, host_target):
         return self.args.test_toolchainbenchmarks
 
+    def _get_test_environment(self, host_target):
+        if platform.system() == 'Darwin':
+            # the resulting binaries would search first in /usr/lib/swift,
+            # we need to prefer the libraries we just built
+            return {'DYLD_LIBRARY_PATH': os.path.join(
+                _get_toolchain_path(host_target, self, self.args),
+                'usr/lib/swift/macosx')}
```

Comment:
Can you remove the '/' and instead use different arguments. That is the canonical way of using join.

Reply:
Fixed in 0bccd8d2e61